### PR TITLE
Annotations for the StorageUnitType

### PR DESCRIPTION
**Pull request by Ruvimbo21Ndoro**

### CODE REVIEW

**Review by Tsireledzo2 — CHANGES_REQUESTED** [review]
the code need changes not right the storsgeunit clas not here and pom file not updated and storage unit type class calling it self which is wrong

File: pom.xml
Comment (Tsireledzo2) [inline pom.xml:1]: The pom file must be updated to the latest onhe

File: src/main/java/za/ac/cput/domain/StorageUnitType.java [hunk 1]
```diff
@@ -11,18 +11,22 @@
...
 
+@Embeddable
 public class StorageUnitType implements Serializable {
 
     private double length;
     private double width;
     private double height;
     private int unitSize;
     private double price;
+    private StorageUnitType storageUnitType;
```
Comment (Tsireledzo2) [inline src/main/java/za/ac/cput/domain/StorageUnitType.java:27]: This is wrong must not be here

**Comment by Ruvimbo21Ndoro** [comment]
Didn't you say l must only do the storageunitType

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: src/main/java/za/ac/cput/domain/StorageUnitType.java [hunk 2]
```diff
@@ -40,22 +44,50 @@ public double getLength() {
...
+
     public double getPrice() {
         return price;
     }
 
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public StorageUnitType getStorageUnitType() {
```
Comment (Tsireledzo2) [inline src/main/java/za/ac/cput/domain/StorageUnitType.java:83]: same as this not suppose to be here

File: src/main/java/za/ac/cput/domain/StorageUnitType.java [hunk 3]
```diff
@@ -40,22 +44,50 @@ public double getLength() {
...
+        return storageUnitType;
+    }
+
+    public void setStorageUnitType(StorageUnitType storageUnitType) {
```
Comment (Tsireledzo2) [inline src/main/java/za/ac/cput/domain/StorageUnitType.java:87]: even this

**Comment by Ruvimbo21Ndoro** [comment]
I followed what you told me to do

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Tsireledzo2** [comment]
use the code here [`here click here please`](https://github.com/Tsireledzo2/Storage-Rental-System/tree/updateStorageUnit)

**Comment by Tsireledzo2** [comment]
When will you update the code?

File: pom.xml
Comment (Tsireledzo2) [inline pom.xml:1]: don't remove anything on the pomfile

File: src/main/java/za/ac/cput/repository/StorageUnitRepository.java [hunk 4]
```diff
@@ -8,11 +8,13 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
 import za.ac.cput.domain.StorageUnit;
 
 import java.util.Set;
 
-public interface StorageUnitRepository extends IRepository <StorageUnit, String> {
+public interface StorageUnitRepository extends JpaRepository<StorageUnit, String> {
+
 
     public Set<StorageUnit> getAll();
```
Comment (Tsireledzo2) [inline src/main/java/za/ac/cput/repository/StorageUnitRepository.java:19]: Change Set to List

**Review by Tsireledzo2 — APPROVED** [review]

**Pull request merged**